### DISCUSSION TITLE
Widget font colour

### DISCUSSION THREAD

**Discussion by Richspiral** [NightscoutFoundation/xDrip#2208] [Q&A]
Hello everyone, I use the widget on my samsung accessibility screen so that the widget displays when my phone is plugged in at night. It's a great feature and means i have a constant display of my sugar levels during the night. However when i go low the font colour changes to red from yellow and is no longer visible on my phone. (This is due to my phone dimming at night time and the red being no longer visible against the black background.) So when i wake up low my sugar level is invisible. This is not helpful in an already confusing situation. There is no where currently to change the widget colour that i can see. Does anyone know a way to switch off the font colour change to red for low blood sugars. Thx!

**Comment by Navid200** [comment]
Do you really mean the widget?

I would like to first make sure we are talking about the same thing.
Would you please have a look at this?
https://navid200.github.io/xDrip/docs/Screensaver.html

Is that what you are talking about?

**Reply by Richspiral** [reply]
No i believe that's the lockscreen settings. I'm talking about the always on display using accessibility settings which i believe uses the widget. 
![Screenshot_20220705-221059_xDrip+](https://user-images.githubusercontent.com/108764983/177417330-bdf074b8-c4f9-4edb-8236-7dc048699127.jpg)

**Reply by Navid200** [reply]
You mean this?  https://github.com/NightscoutFoundation/xDrip/issues/1625

**Reply by Richspiral** [reply]
Yes, that's it. Am i incorrect in thinking that's it's using the widget?

**Reply by Navid200** [reply]
I'm sorry I didn't know you were talking about that.

If you experience any problems, considering that is a feature that needs testers with an open issue related, I suggest you look through the linked issue to see if anyone else has mentioned it.  If not, I suggest you mention it there in that issue.

**Reply by Richspiral** [reply]
No probs and will do... Thanks for the sign posting! 👍

**Reply by Der-Schubi** [reply]
I've done a PR for this: https://github.com/NightscoutFoundation/xDrip/pull/2212
